docs/.vitepress/theme/custom.css

```diff
@@ -2,10 +2,6 @@
 /* [vp] = vitepress, tokens used in the documentation interface itself */
 /* [c] = color, omit if not a color */
 
-/* Todo: Clean everything up to follow this prefix  */
```

Comment:
Removed this comment after discussing it with @Skadefryd:
"For tidiness I would prob suggest they either follow the prefix OR they use something that follow that prefix,   like --vp-my3rdpartything:  var(--vp-c-brand-1). Since this is a ‘we make this site once’ and rarely touch it again, it's not worth it to spend too much time on it though.  (And that might change if we do a redesign)"